.github/workflows/macos_python.yml

```diff
@@ -25,9 +25,6 @@ jobs:
     runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
-      with:
-        python-version: 3.7
```

Comment:
Using this action is recommended by Github, see [here](https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#specifying-a-python-version). Maybe we can bump the Python version instead of relying on system Python?